lib/compat/wordpress-6.0/theme.json

```diff
@@ -185,6 +185,85 @@
 			],
 			"text": true
 		},
+		"layout": {
+			"definitions": {
+				"default": {
+					"name": "default",
+					"slug": "flow",
+					"className": "is-layout-flow",
+					"baseStyles": [
+						{
+							"selector": " > .alignleft",
+							"rules": {
+								"float": "left",
+								"margin-inline-start": "0",
+								"margin-inline-end": "2em"
+							}
+						},
+						{
+							"selector": " > .alignright",
+							"rules": {
+								"float": "right",
+								"margin-inline-start": "2em",
+								"margin-inline-end": "0"
+							}
+						},
+						{
+							"selector": " > .aligncenter",
+							"rules": {
+								"margin-left": "auto !important",
+								"margin-right": "auto !important"
+							}
+						}
+					],
+					"blockGapStyles": [
```

Comment:
Oh, this is a great point too — part of the trouble is that the setting began as `blockGap` and later on the label was changed to `Block spacing`. With blocks out in the wild still using the `blockGap` attribute to store the values, I thought it'd probably make sense to try to keep it aligned... but, what if we called the key her `spacingStyles` so that we can avoid using the word `block`?

Reply:
Was `blockGap` already being used outside of a flex layout? It might make sense to keep it then. Otherwise `spacingStyles` sounds good!

Reply:
Currently, I believe `blockGap` is only really used for the direct value in block attributes, and I think it makes for a clearer name if we use `spacingStyles` for the layout definitions, as it's a bit broader than just the gap value, like mention. I've updated the naming in https://github.com/WordPress/gutenberg/pull/40875/commits/71c7288f92d689d91e084e78f786c9776a51cd30.

I'll resolve this comment thread, but feel free to unresolve if you think it should be tweaked further 🙂